conseil-common/src/main/scala/tech/cryptonomic/conseil/common/config/Platforms.scala

```diff
@@ -10,32 +10,31 @@ object Platforms {
 
     /** a name usually used in configurations to identify the platform */
     def name: String
-
-    /** the type of underlying configuration, specific for the platform */
-    type ConfigurationType <: PlatformConfiguration
   }
 
+  /** Represents blockchain platform for Tezos */
   case object Tezos extends BlockchainPlatform {
     val name = "tezos"
-    type ConfigurationType = TezosConfiguration
   }
 
+  /** Represents blockchain platform for Bitcoin */
   case object Bitcoin extends BlockchainPlatform {
     val name = "bitcoin"
-    type ConfigurationType = BitcoinConfiguration
   }
 
-  case class UnknownPlatform(name: String) extends BlockchainPlatform {
-    type ConfigurationType = UnknownPlatformConfiguration
+  /** Represents blockchain platform, which is use only during arguments parsing, for better error handling */
+  case object Unknown extends BlockchainPlatform {
```

Comment:
Didn't you remove this?

Reply:
Yes I did, but then I changed the way of reading command-line arguments and for this specific reason (in here: tech.cryptonomic.conseil.indexer.config.LorreAppConfig.ArgumentsConfig) default value is required. I could use option tho, but this is not optional. I could use `null` as default value, but I'm not sure if that is a better thing to do.

I did this change (String -> BlockchainPlatform), because previously we allowed unknown platforms to go really deep in the code, when, in my opinion this should be handled as soon as possible (and it handled at reading command-line arguments) for Lorre process only.

I can actually change default value to any platform we have (Tezos even), cause it does not really matter. It will be overwritten anyway during parsing process.

Reply:
We can leave the unknown, or at this point we can rename it as undefined?

In any case, I just pointed that out, because I remembered you removing this case and I thought you forgot it somehow when re-opening the PR.

Reply:
Yea. I mean, I just realized that there is no really point in having it. At case class initialization we can put whatever we want, it will be changed either way, so I just put `Tezos` instead of `Unknown` and that was the only place, where this case object was used. I feel better without having it :)